build/commands/lib/util.js

```diff
@@ -284,6 +284,12 @@ const util = {
       path.join(config.srcDir, 'ui', 'webui', 'resources', 'images',
           'chrome_logo_dark.svg')])
 
+    // Override Password Manager icons
```

Comment:
This is really not a great way to handle this stuff and we need to stop using this. In this particular case you can do a chromium_src override for password_manager_ui.cc 
```
#define SetupWebUIDataSource(...) SetupWebUIDataSource(__VA_ARGS__); source->AddResourcePath("images/password_manager_logo.svg",
                          IDR_BRAVE_PASSWORD_MANAGER_LOGO);
```
where `IDR_BRAVE_PASSWORD_MANAGER_LOGO` goes in `brave_unscaled_resources.grd`.

Reply:
fyi - this does the same thing as the google chrome branding in this method
```
#if BUILDFLAG(GOOGLE_CHROME_BRANDING)
  // Overwrite ubranded logo for Chrome-branded builds.
  source->AddResourcePath("images/password_manager_logo.svg",
                          IDR_CHROME_PASSWORD_MANAGER_LOGO);

  // This path is used in the manifest of the PasswordManager web app
  // (chrome/browser/resources/password_manager/
  // chrome_branded_manifest.webmanifest).
  source->AddResourcePath("images/password_manager_pwa_icon.svg",
                          IDR_CHROME_PASSWORD_MANAGER_PWA_ICON);
#endif
```

Reply:
Cool, thanks @bridiver - I was just cargo culting the other stuff :)